tests/networks/networks_test.py

```diff
@@ -70,10 +70,19 @@ def test_shared_networks(self, system: Any) -> None:
             "agent_1": "agent",
             "agent_2": "agent",
         }
-        networks = system.make_default_networks(  # type: ignore
-            environment_spec=environment_spec,  # type: ignore
-            agent_net_keys=agent_net_keys,
-        )
+
+        if system == mad4pg:
+            networks = system.make_default_networks(  # type: ignore
+                environment_spec=environment_spec,  # type: ignore
+                agent_net_keys=agent_net_keys,
+                vmin=-150,
```

Comment:
Should we maybe add `**kwargs` to the `make_default_networks` so that we don't have to have if statements like this?

Reply:
Yes so **kwargs is what we had. But I removed the kwargs for vmin and vmax. The reason for this being that vmin and vmax is environment specific and assigning any default value would not really make sense. I am just afraid that the user might just accept the default value and this would significantly hamper their training if the default is values are not correct. So I changed it to force the user to specify it. Let me know what you think.

Reply:
Would it not work to have `**kwargs` and then simply not have defaults for vmin and vmax in the mad4pg default networks function?

Reply:
Thanks. This was resolved in a call.

Reply:
Just checking if this will change or stay like this?

Reply:
We decided to keep it as is.